### ISSUE TITLE
Regression after releasing NewRelic Teraform provider 3.25.1

### ISSUE THREAD

**Issue by pattrykks** [newrelic/terraform-provider-newrelic#2418]
Regression was detected after updating Terraform NewRelic provider 3.25.0 -> 3.25.1. I suspect the problem is the following commit: https://github.com/newrelic/terraform-provider-newrelic/pull/2402/files

# What is the behaviour/problem?
There are NewRelic Terraform resources for which  `violation_close_timer` is optional. In case of specific configurations using it is even not allowed (validation preventing defining such resources exists). Example `newrelic_alert_condition` with `type`: `apm_app_metric` and `condition_scope`: `application`. During trial to `terraform apply` such resources when user will not provide resource at all in Terraform code or will set `violation_close_timer` to null then `default=72` is applied which cause resource validation is failing

# What kind of resource is affected. In my case:
* `newrelic_alert_condition`

# What is error message
```
╷
│ Error: violation_close_timer only supported for apm_app_metric when condition_scope = 'instance'
│ 
│   with module.newrelic.module.app_alerts.newrelic_alert_condition.response_time_web["blahblah"],
│   on modules/newrelic/modules/app_alerts/main.tf line 170, in resource "newrelic_alert_condition" "response_time_web":
│  170: resource "newrelic_alert_condition" "response_time_web" {
```

# How resources are defined
```
resource "newrelic_alert_condition" "appdex" {
  for_each  =  toset(['<MY_APP_1>'])
  policy_id = '<POLICY_ID>'

  name            = "<ALERT_NAME>"
  type            = "apm_app_metric"
  entities        = [<ENTITIES>]
  metric          = "apdex"
  condition_scope = "application"

  term {
    duration      = 5
    operator      = "below"
    priority      = "warning"
    threshold     = "0.85"
    time_function = "all"
  }

  term {
    duration      = 5
    operator      = "below"
    priority      = "critical"
    threshold     = "0.7"
    time_function = "all"
  }
}
```

```
resource "newrelic_alert_condition" "response_time_web" {
  for_each = toset([<MY_APP_1>])
  policy_id = <POLICY_ID>

  name            = "<ALERT_NAME>"
  type            = "apm_app_metric"
  entities        = [<ENTITY>]
  metric          = "response_time_web"
  condition_scope = "application"

  term {
    duration      = 15
    operator      = "above"
    priority      = "critical"
    threshold     = "1.0"
    time_function = "all"
  }
}
```

# Additional info
When using old pipelines with old version of NewRelic Terraform provider (`3.25.0`) everything is working as expected

**Comment by pranav-new-relic** [comment]
Hi @pattrykks - thank you for reporting this issue. I've been able to reproduce it - and it is partly because of the change made in the commit you've attached. `violation_close_timer` needs to default to 72 only in a certain set of cases, and not all, which is what is showing an API validation issue upon running this code with v3.25.1. We'll get this corrected at the earliest.

**Comment by pattrykks** [comment]
Thank you so much for such instant answer. And sorry for my chaotic description

**Comment by pranav-new-relic** [comment]
Hi @pattrykks - we've reverted the faulty change and pushed a new release, v3.25.2. You can now try creating the Alert Condition specified in the description with this new version, and let me know if the issue persists. Thanks for reporting!

**Comment by pattrykks** [comment]
Hi @pranav-new-relic many thanks. We already migrated to newest version. All looks good on our side now.

**Comment by pranav-new-relic** [comment]
Cool, thanks for the update :)

**Comment by pranav-new-relic** [comment]
> Hi @pattrykks - we've reverted the faulty change and pushed a new release, v3.25.2. You can now try creating the Alert Condition specified in the description with this new version, and let me know if the issue persists. Thanks for reporting!

@Tuzosdaniel12 you can now try creating an alert condition with the configuration you've reported in the other issue - as specified above, the fix is out with v3.25.2 of the provider.
